CONTRIBUTORS.md

```diff
@@ -308,6 +308,7 @@ Created as part of the release process.
 + Wilhelmina Drengwitz
 + Win Wang
 + Wolfram Arnold
++ Worker Pants (Pantsbuild GitHub Automation Bot)
```

Comment:
Not what was intended, but I guess we can leave it in there? Or should we script around it? @thejcannon your thoughts?

Reply:
Previous thread on this topic: https://github.com/pantsbuild/pants/pull/19933#discussion_r1335293568

Reply:
🙏